addons/mail/static/src/new/core/persona_service.js

```diff
@@ -0,0 +1,68 @@
+/** @odoo-module */
+
+import { Persona } from "@mail/new/core/persona_model";
+import { createLocalId } from "./thread_model.create_local_id";
+
+export class PersonaService {
+    constructor(env, services) {
+        this.env = env;
+        this.rpc = services.rpc;
+        /** @type {import("@mail/new/core/store_service").Store} */
+        this.store = services["mail.store"];
+    }
+
+    async updateGuestName(guest, name) {
+        await this.rpc("/mail/guest/update_name", {
+            guest_id: guest.id,
+            name,
+        });
+    }
```

Comment:
Just to make sure: name is updated locally with return of RPC?
Maybe worth a comment if that is the case.

Reply:
This function is called when the current guest changes its name. It's immediately changed in store, and this function is just to apply changes server-side (+ broadcast to other members).